docs/samples/v1beta1/torchserve/pvpod.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: model-store-pod
```

Comment:
@yuzliu @ifilonenko this probably can be done in our new model puller in future where it can pull the model from remote storage and cache on PVC.